src/services/DocumentSelectionRequestProcessor.ts

```diff
@@ -99,6 +104,13 @@ export class DocumentSelectionRequestProcessor {
   		throw new AppError(HttpCodesEnum.BAD_REQUEST, "Missing details in SESSION or PERSON IDENTITY tables");
   	}
 
+  	// Reject the request when session store does not contain email, familyName or GivenName fields
+  	const data = this.validationHelper.isPersonDetailsValid(personDetails.emailAddress, personDetails.name);
+  	if (data.errorMessage.length > 0) {
+  		this.logger.error( data.errorMessage + " in the PERSON IDENTITY table", { messageCode : data.errorMessageCode });
+  		return new Response(HttpCodesEnum.SERVER_ERROR, data.errorMessage + " in the PERSON IDENTITY table");
```

Comment:
nit: Why not just return "in the PERSON IDENTITY table" in the data.errorMessage too?